sdk/cosmos/ci.spark.databricks.yml

```diff
@@ -0,0 +1,53 @@
+jobs:
+- job: Spark_CosmosDB_Databricks
```

Comment:
once this merge how this CI can be triggered?

is there a command for that?

Reply:
We should be able to trigger it with `/azp run <some-name>`